bodhi/server/models.py

```diff
@@ -3706,6 +3708,9 @@ def comment_on_test_gating_status_change(target, value, old, initiator):
         """
         Place comment on the update when ``test_gating_status`` changes.
 
+        Only notify the users by email if the new status is not in ``waiting``,
```

Comment:
This looks like it's saying the opposite from what it's doing. Shouldn't `not` be removed? Also, what about the `waiting` state, it's not in the code below.

Reply:
sorry missed to update the comment in both sections, will adjust